truffle.js

```diff
@@ -0,0 +1,2 @@
+module.exports = require("@aragon/os/truffle-config")
```

Comment:
Seems quite odd to have a truffle config here, and to require `aragonOS` for it, but :).

Reply:
I feel that, but previously I had a local link to the aragonOS directory in my machine which wasn't great for everyone to be able to easily publish. If you have a better idea I will gladly change it :)

Reply:
We can move the truffle config to be its own aragon-namespaced package later :).